test/org/loklak/api/search/EventBriteCrawlerServiceTest.java

```diff
@@ -26,119 +27,117 @@ public void apiPathTest() {
-        assertEquals("Facebook", fbname);
-        assertEquals("1", fbid);
-        assertEquals("Twitter", twname);
-        assertEquals("2", twid);
-        assertEquals("2030-05-25T22:00:00", end_time);
-        assertEquals("It is a test event made only for testing purpose. Apologies for trouble.", description);
-        assertEquals("public", privacy);
-        assertEquals("", type);
-        assertEquals("https://www.eventbrite.com/e/testevent-tickets-46017636991#tickets", ticket_url);
-        assertEquals("https://www.eventbrite.com/e/testevent-tickets-46017636991", event_url);
-        assertEquals("2030-05-18T19:00:00", start_time);
-        assertEquals("TestEvent by Saurabh Srivastava", name);
-        assertEquals("https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F44763103%2F256428075899%2F1%2Foriginal.jpg?w=1000&auto=compress&rect=0%2C67%2C400%2C200&s=9018130444506b1cffa6d4bd11b0fddb", logo);
-        assertEquals("", topic);
-        assertEquals("46017636991", orgId);
-        assertEquals("aurabh Srivastava", organizer_name);
-        assertEquals("completed", state);
-        assertTrue("Not equals", latDiff == 0);
-        assertTrue("Not equals", lonDiff == 0);
-        assertNotNull(location_name);
+            assertEquals("1", id);
+            assertEquals("", email);
+            assertEquals("https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F44763103%2F256428075899%2F1%2Foriginal.jpg?w=1000&auto=compress&rect=0%2C67%2C400%2C200&s=9018130444506b1cffa6d4bd11b0fddb", background_url);
+            assertEquals("Facebook", fbname);
+            assertEquals("1", fbid);
+            assertEquals("Twitter", twname);
+            assertEquals("2", twid);
+            assertEquals("2030-05-25T22:00:00", end_time);
+            assertEquals("It is a test event made only for testing purpose. Apologies for trouble.", description);
+            assertEquals("public", privacy);
+            assertEquals("", type);
+            assertEquals("https://www.eventbrite.com/e/testevent-tickets-46017636991#tickets", ticket_url);
+            assertEquals("https://www.eventbrite.com/e/testevent-tickets-46017636991", event_url);
+            assertEquals("2030-05-18T19:00:00", start_time);
+            assertEquals("TestEvent by Saurabh Srivastava", name);
+            assertEquals("https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F44763103%2F256428075899%2F1%2Foriginal.jpg?w=1000&auto=compress&rect=0%2C67%2C400%2C200&s=9018130444506b1cffa6d4bd11b0fddb", logo);
+            assertEquals("", topic);
+            assertEquals("46017636991", orgId);
+            assertEquals("aurabh Srivastava", organizer_name);
+            assertEquals("completed", state);
+            assertTrue("Not equals", latDiff == 0);
+            assertTrue("Not equals", lonDiff == 0);
+            assertNotNull(location_name);
+        } catch (Exception e) {
```

Comment:
Do not catch so broad exceptions either. If we are expecting to ignore network errors only, just catch corresponding errors.